src/chrome/content/rules/Financial_Times.xml

```diff
@@ -40,7 +40,6 @@
 -->
 <ruleset name="Financial Times (partial)">
 	<target host="ft.com" />
-	<target host="www.ft.com" />
```

Comment:
One question: Why only www.ft.com is removed when ft.com itself is preloaded?

Reply:
It's missing the `includeSubDomains` directive.

See https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#hsts-preloaded-rules and https://securityheaders.com/?q=https%3A%2F%2Fft.com%2F